db/migrations/20170713094702_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.raw('UPDATE movies SET name = title');
```

Comment:
since we changed the create endpoint to delete the title from the payload, we only want to perform this when title is not null